lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -2472,6 +2497,18 @@ void HdVP2Mesh::_UpdateDrawItem(
 }
 
 void HdVP2Mesh::_HideAllDrawItems(const TfToken& reprToken)
+{
+    auto hideDrawItem = [this](HdVP2DrawItem::RenderItemData& renderItemData) {
+        renderItemData._enabled = false;
+        _delegate->GetVP2ResourceRegistry().EnqueueCommit(
+            [&]() { renderItemData._renderItem->enable(false); });
+    };
+
+    _ForEachRenderItemInRepr(reprToken, hideDrawItem);
+}
+
+template <typename Func>
+void HdVP2Mesh::_ForEachRenderItemInRepr(const TfToken& reprToken, Func func)
```

Comment:
I think this could more to the parent class once the clean up gets merged.

Reply:
Yeah. I moved this function to the parent class so this will have to be propely merged.